applications/ChimeraApplication/chimera_application_variables.h

```diff
@@ -0,0 +1,41 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Authors:        Aditya Ghantasala, https://github.com/adityaghantasala
+// 					Navaneeth K Narayanan
+//					Rishith Ellath Meethal
+//
+
+#if !defined(KRATOS_CHIMERA_APPLICATION_VARIABLES_H_INCLUDED)
+#define KRATOS_CHIMERA_APPLICATION_VARIABLES_H_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "includes/variables.h"
```

Comment:
```suggestion
#include "includes/variable.h"
```
Otherwise you add all the variables from the Core even if you don't need them!
(see template app)

Reply:
resolved !